.github/workflows/build-extensions.yml

```diff
@@ -370,26 +398,50 @@ jobs:
     strategy:
       matrix:
         include:
-          - name: WASI-Crypto
-            tar_name: wasi_crypto
-            system: MacOS 11
+          - name: WASI-NN OpenVINO
+            tar_name: wasi_nn-openvino
+            backend: OpenVINO
             host_runner: macos-11
             darwin_version: darwin_20
-            output_path: plugins/wasi_crypto
-            output_bin: libwasmedgePluginWasiCrypto.dylib
-            test_path: test/plugins/wasi_crypto
-            test_bin: wasiCryptoTests
+            output_path: plugins/wasi_nn
+            output_bin: libwasmedgePluginWasiNN.dylib
+            test_path: test/plugins/wasi_nn
+            test_bin: wasiNNTests
+            cmake_plugin_option: "-DWASMEDGE_PLUGIN_WASI_NN_BACKEND=OpenVINO"
             build_type: Release
-          - name: WASI-Crypto
-            tar_name: wasi_crypto
-            system: MacOS 12
+          - name: WASI-NN OpenVINO
+            tar_name: wasi_nn-openvino
+            backend: OpenVINO
             host_runner: macos-12
             darwin_version: darwin_21
-            output_path: plugins/wasi_crypto
-            output_bin: libwasmedgePluginWasiCrypto.dylib
-            test_path: test/plugins/wasi_crypto
-            test_bin: wasiCryptoTests
+            output_path: plugins/wasi_nn
+            output_bin: libwasmedgePluginWasiNN.dylib
+            test_path: test/plugins/wasi_nn
+            test_bin: wasiNNTests
+            cmake_plugin_option: "-DWASMEDGE_PLUGIN_WASI_NN_BACKEND=OpenVINO"
             build_type: Release
+          # - name: WASI-Crypto
```

Comment:
If you want to comment out something, you must leave the reason in the comments.